crates/modelardb_server/tests/integration_test.rs

```diff
@@ -369,102 +332,92 @@ impl TestContext {
-    fn flush_data_to_disk(&mut self) {
+    async fn flush_data_to_disk(&mut self) {
         let action = Action {
             r#type: "FlushMemory".to_owned(),
             body: Bytes::new(),
         };
 
-        self.runtime.block_on(async {
-            self.client.do_action(Request::new(action)).await.unwrap();
-        })
+        self.client.do_action(Request::new(action)).await.unwrap();
     }
 
     /// Execute a query against the server through the `do_get()` method and return it.
-    fn execute_query(&mut self, query: String) -> Result<RecordBatch, Box<dyn Error>> {
-        self.runtime.block_on(async {
-            // Execute query.
-            let ticket = Ticket {
-                ticket: query.into(),
-            };
-            let mut stream = self.client.do_get(ticket).await?.into_inner();
-
-            // Get schema of result set.
-            let flight_data = stream.message().await?.ok_or("No time series received.")?;
-            let schema = Arc::new(Schema::try_from(&flight_data)?);
-
-            // Get data in result set.
-            let mut query_result = vec![];
-            while let Some(flight_data) = stream.message().await? {
-                let dictionaries_by_id = HashMap::new();
-                let record_batch = utils::flight_data_to_arrow_batch(
-                    &flight_data,
-                    schema.clone(),
-                    &dictionaries_by_id,
-                )?;
-                query_result.push(record_batch);
-            }
+    async fn execute_query(&mut self, query: String) -> Result<RecordBatch, Box<dyn Error>> {
+        // Execute query.
+        let ticket = Ticket {
+            ticket: query.into(),
+        };
+        let mut stream = self.client.do_get(ticket).await?.into_inner();
+
+        // Get schema of result set.
+        let flight_data = stream.message().await?.ok_or("No time series received.")?;
+        let schema = Arc::new(Schema::try_from(&flight_data)?);
+
+        // Get data in result set.
+        let mut query_result = vec![];
+        while let Some(flight_data) = stream.message().await? {
+            let dictionaries_by_id = HashMap::new();
+            let record_batch = utils::flight_data_to_arrow_batch(
+                &flight_data,
+                schema.clone(),
+                &dictionaries_by_id,
+            )?;
+            query_result.push(record_batch);
+        }
 
-            if query_result.is_empty() {
-                Ok(RecordBatch::new_empty(schema))
-            } else {
-                // unwrap() is used as ? makes the return type Result<RecordBatch, ArrowError>>.
-                Ok(compute::concat_batches(&query_result[0].schema(), &query_result).unwrap())
-            }
-        })
+        if query_result.is_empty() {
+            Ok(RecordBatch::new_empty(schema))
+        } else {
+            // unwrap() is used as ? makes the return type Result<RecordBatch, ArrowError>>.
```

Comment:
Maybe delete the `unwrap()` comment as it does not seem to be used elsewhere in the file.

Reply:
Fixed.